### DISCUSSION TITLE
Shipping menu is not available when install manually

### DISCUSSION THREAD

**Discussion by mmrtonmoybd** [lunarphp/lunar#1994] [Q&A] [answered]
Hi everyone, 
### For manually install
When I installed manually by following this [instruction process](https://docs.lunarphp.io/core/installation.html) all are perfectly done but "Shipping " menu is not showing in lunar admin panel. 
Note: I also install these two packages lunarphp/stripe and lunarphp/table-rate-shipping . Also I manully [seed](https://[raw.githubusercontent.com/lunarphp/livewire-starter-kit/refs/heads/main/database/seeders/ShippingSeeder.php](https://raw.githubusercontent.com/lunarphp/livewire-starter-kit/refs/heads/main/database/seeders/ShippingSeeder.php)) this but at the end do not showing shipping menu. 
**Manually installed screenshot**
![image](https://github.com/user-attachments/assets/cf856c8f-ab0a-4c14-a92b-a172039c1708)

### For starter kit:
But when install lunar livewire starter kit it show "Shipping" menu in admin panel by default. 
**Screenshot starter kit**
![image](https://github.com/user-attachments/assets/5d3651ae-4aa2-447a-a156-3adb23da5ec1)

How I showing this menu in manually installed process ?

**Comment by mmrtonmoybd** [comment, marked as answer]
After wasting many hours find the solution. 
In App\Providers\AppServiceProvider.php register method :
` LunarPanel::panel(
            fn ($panel) => $panel->plugins([
                new ShippingPlugin,
            ])
        )
            ->register();`

**Reply by alecritson** [reply]
Hi @mmrtonmoybd When using Lunar addons it's always recommended to look through the README.md file for it as there will likely be instructions on how to install. Thanks.
